Test log noise because of references to JBoss LogManager

Running the 1brc test now gives a lot of noise about the LogManager. I think this only happens in the IDE, and is noisy but non-fatal. 

```
/Library/Java/JavaVirtualMachines/temurin-21.jdk/Contents/Home/bin/java -ea -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Dmaven.home=/Applications/IntelliJ IDEA.app/Contents/plugins/maven/lib/maven3 -Dmaven.repo=/Users/holly/.m2/repository -Dmaven.repo.local=/Users/holly/.m2/repository -Djava.io.tmpdir=/Users/holly/Code/demos/rockstar/bon-jova/quarkus-bon-jova-rockstar/compiler/target -Didea.test.cyclic.buffer.size=1048576 -javaagent:/Applications/IntelliJ IDEA.app/Contents/lib/idea_rt.jar=58149:/Applications/IntelliJ IDEA.app/Contents/bin -Dfile.encoding=UTF-8 -Dsun.stdout.encoding=UTF-8 -Dsun.stderr.encoding=UTF-8 -classpath /Users/holly/.m2/repository/org/junit/platform/junit-platform-launcher/1.10.1/junit-platform-launcher-1.10.1.jar:/Applications/IntelliJ IDEA.app/Contents/lib/idea_rt.jar:/Applications/IntelliJ IDEA.app/Contents/plugins/junit/lib/junit5-rt.jar:/Applications/IntelliJ IDEA.app/Contents/plugins/junit/lib/junit-rt.jar:/Users/holly/Code/demos/rockstar/bon-jova/quarkus-bon-jova-rockstar/compiler/target/test-classes:/Users/holly/Code/demos/rockstar/bon-jova/quarkus-bon-jova-rockstar/compiler/target/classes:/Users/holly/.m2/repository/io/quarkus/gizmo/gizmo/1.7.0/gizmo-1.7.0.jar:/Users/holly/.m2/repository/org/ow2/asm/asm/9.6/asm-9.6.jar:/Users/holly/.m2/repository/org/ow2/asm/asm-util/9.6/asm-util-9.6.jar:/Users/holly/.m2/repository/org/ow2/asm/asm-tree/9.6/asm-tree-9.6.jar:/Users/holly/.m2/repository/org/ow2/asm/asm-analysis/9.6/asm-analysis-9.6.jar:/Users/holly/.m2/repository/io/smallrye/jandex/3.1.6/jandex-3.1.6.jar:/Users/holly/.m2/repository/io/quarkus/gizmo/gizmo/1.7.0/gizmo-1.7.0-tests.jar:/Users/holly/.m2/repository/org/antlr/antlr4-runtime/4.13.0/antlr4-runtime-4.13.0.jar:/Users/holly/.m2/repository/org/junit/jupiter/junit-jupiter-engine/5.10.0/junit-jupiter-engine-5.10.0.jar:/Users/holly/.m2/repository/org/junit/platform/junit-platform-engine/1.10.1/junit-platform-engine-1.10.1.jar:/Users/holly/.m2/repository/org/opentest4j/opentest4j/1.3.0/opentest4j-1.3.0.jar:/Users/holly/.m2/repository/org/junit/platform/junit-platform-commons/1.10.1/junit-platform-commons-1.10.1.jar:/Users/holly/.m2/repository/org/junit/jupiter/junit-jupiter-api/5.10.1/junit-jupiter-api-5.10.1.jar:/Users/holly/.m2/repository/org/apiguardian/apiguardian-api/1.1.2/apiguardian-api-1.1.2.jar com.intellij.rt.junit.JUnitStarter -ideVersion5 -junit5 io.quarkiverse.bonjova.compiler.integration.IntegrationTests,shouldRunOneBillionRowChallenge
Could not load Logmanager "org.jboss.logmanager.LogManager"
java.lang.ClassNotFoundException: org.jboss.logmanager.LogManager
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at java.logging/java.util.logging.LogManager$1.run(LogManager.java:239)
	at java.logging/java.util.logging.LogManager$1.run(LogManager.java:223)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:319)
	at java.logging/java.util.logging.LogManager.<clinit>(LogManager.java:222)
	at java.logging/java.util.logging.Logger.demandLogger(Logger.java:650)
	at java.logging/java.util.logging.Logger.getLogger(Logger.java:720)
	at java.logging/java.util.logging.Logger.getLogger(Logger.java:703)
	at org.junit.platform.commons.logging.LoggerFactory$DelegatingLogger.<init>(LoggerFactory.java:80)
	at org.junit.platform.commons.logging.LoggerFactory.getLogger(LoggerFactory.java:51)
	at org.junit.platform.launcher.core.LauncherConfigurationParameters.<clinit>(LauncherConfigurationParameters.java:41)
	at org.junit.platform.launcher.core.LauncherFactory.create(LauncherFactory.java:129)
	at org.junit.platform.launcher.core.LauncherFactory.create(LauncherFactory.java:112)
	at com.intellij.junit5.JUnit5IdeaTestRunner.createListeners(JUnit5IdeaTestRunner.java:34)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:232)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:55)

org.opentest4j.AssertionFailedError
	at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:46)
	at org.junit.jupiter.api.Assertions.fail(Assertions.java:161)
	at io.quarkiverse.bonjova.compiler.util.FileLauncher.compileAndLaunch(FileLauncher.java:80)
	at io.quarkiverse.bonjova.compiler.integration.IntegrationTests.shouldRunOneBillionRowChallenge(IntegrationTests.java:54)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: org.opentest4j.AssertionFailedError: Execution did not complete successfully. Output was:
Exception in thread "main" java.lang.NoClassDefFoundError: boolean
	at rock12196129597320738163.main(Unknown Source)
Caused by: java.lang.ClassNotFoundException: boolean
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	... 1 more
 ==> expected: <0> but was: <1>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:563)
	at io.quarkiverse.bonjova.compiler.util.FileLauncher.launch(FileLauncher.java:61)
	at io.quarkiverse.bonjova.compiler.util.FileLauncher.compileAndLaunch(FileLauncher.java:78)
	... 4 more
```

I briefly included the logger in my bytecode generator, then took it out, so I'm not totally sure where the failures are coming from.

--- a/quarkus-bon-jova-rockstar/compiler/pom.xml
+++ b/quarkus-bon-jova-rockstar/compiler/pom.xml
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
@@ -30,6 +31,13 @@
             <artifactId>gizmo</artifactId>
             <type>test-jar</type>
             <version>${gizmo.version}</version>
+        </dependency>
+        <!-- Needed because the quarkiverse parent sets a logmanager in its surefire configuration, and we get annoying logger not found messages otherwise -->
+        <dependency>
+            <groupId>org.jboss.logmanager</groupId>
+            <artifactId>jboss-logmanager</artifactId>
+            <version>3.0.5.Final</version>
+            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.antlr</groupId>

